dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -402,7 +433,11 @@ internal open class RumViewScope(
                 crash = ViewEvent.Crash(crashCount),
                 longTask = ViewEvent.LongTask(longTaskCount),
                 customTimings = timings,
-                isActive = !stopped
+                isActive = !stopped,
+                cpuTicksCount = cpuTicks,
+                cpuTicksPerSecond = cpuTicks?.let { (it * ONE_SECOND_NS) / updatedDurationNs },
```

Comment:
I would extract this in a separated method

Reply:
I could but I don't really see the point of it since this computation is only used here 🤷